### PR TITLE
Refine general documentation

### PR DESCRIPTION
**Pull request by qwe661234**

### CODE REVIEW

File: Document.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 The code section enclosed by `batch_start()` and `batch_flush()` is called batching segment. It can appear more than one time in a single application. Compared with typical syscalls, ESCA eliminates mode switches in batching segments by decoupling syscalls. Instead of switching to the kernel or executing the corresponding service routine, syscalls in batching segment only record their syscall ID and arguments in the shared table.After batch_flush is called, ESCA finally switches to kernel mode, executes all syscalls in the shared table, and then switches back to user mode.
 ## Typical system call flow
 ### Overview
-![](https://i.imgur.com/zEXqLKH.png)
+![image](assets/systemcall_flow.png)
```
Comment (jserv) [inline Document.md]: Rename to `assets/syscall-flow.png`.

Comment (qwe661234) [reply Document.md]: done

File: Document.md [hunk 2]
```diff
@@ -170,5 +170,4 @@ __attribute__((constructor)) static void setup(void)
 ## Reference 
 * [GitHub: ESCA](https://github.com/eecheng87/ESCA)
```
Comment (jserv) [inline Document.md]: Remove "GitHub: ESCA" because we are now reading this page.

File: Document.md [hunk 3]
```diff
@@ -170,5 +170,4 @@ __attribute__((constructor)) static void setup(void)
...
 * [ESCA thesis](https://eecheng87.github.io/ESCA/main.pdf)
```
Comment (jserv) [inline Document.md]: Drop "ESCA thesis" because it was mentioned in top-level `README.md`.

File: Document.md [hunk 4]
```diff
@@ -169,6 +169,4 @@ __attribute__((constructor)) static void setup(void)
 
 ## Reference
```
Comment (jserv) [inline Document.md:170]: Check ESCA paper to decide the relevant materials to appear in "Reference" section.
Also, check [What is the Difference Between Thesis and Research Paper](https://pediaa.com/what-is-the-difference-between-thesis-and-research-paper/)
> thesis is a long academic paper that typically serves as the final project for a university degree, while research paper is a piece of academic writing on a particular topic.

That is, there is a paper about ESCA, but there is no such thesis.

File: Document.md [hunk 5]
```diff
@@ -3,7 +3,7 @@
 The code section enclosed by `batch_start()` and `batch_flush()` is called batching segment. It can appear more than one time in a single application. Compared with typical syscalls, ESCA eliminates mode switches in batching segments by decoupling syscalls. Instead of switching to the kernel or executing the corresponding service routine, syscalls in batching segment only record their syscall ID and arguments in the shared table.After batch_flush is called, ESCA finally switches to kernel mode, executes all syscalls in the shared table, and then switches back to user mode.
 ## Typical system call flow
 ### Overview
-![](https://i.imgur.com/zEXqLKH.png)
+![image](assets/syscall-flow.png)
```
Comment (jserv) [inline Document.md:6]: This diagram IS NOT drawn by you, and you MUST mention the source and credit.

Comment (jserv) [reply Document.md:6]: @eecheng87, You must be aware of fair use guidelines for outside materials.

Comment (eecheng87) [reply Document.md:6]: @qwe661234 , as an alternative, you can use the figure I drawn in Page 6 in this [slide](https://docs.google.com/presentation/d/1aB-t-8wDnsQ07VTGmQd6F5bKZYseZlj8CJ80rJqsj_4/edit?usp=sharing).

Comment (qwe661234) [reply Document.md:6]: > @qwe661234 , as an alternative, you can use the figure I drawn in Page 6 in this [slide](https://docs.google.com/presentation/d/1aB-t-8wDnsQ07VTGmQd6F5bKZYseZlj8CJ80rJqsj_4/edit?usp=sharing).

Thanks, I changed image to the figure in Page 6 in this [slide](https://docs.google.com/presentation/d/1aB-t-8wDnsQ07VTGmQd6F5bKZYseZlj8CJ80rJqsj_4/edit#slide=id.p)

File: Document.md (hunk 4 above)
Comment (qwe661234) [reply Document.md:170]: Ok, I added the relevant materials in "Reference" section and I will check the difference between thesis and research paper later.

File: Document.md (hunk 3 above)
Comment (qwe661234) [reply Document.md]: done

File: Document.md (hunk 2 above)
Comment (qwe661234) [reply Document.md]: done

**Pull request merged**